tests/expressions_case/tests.py

```diff
@@ -1165,6 +1165,16 @@ def test_m2m_reuse(self):
             lambda x: x[1:]
         )
 
+    def test_empty_q_object(self):
+        """A Q object with an empty condition should be rejected as the
+        conditional expression in a Case()."""
+        with self.assertRaisesMessage(ValueError, "Empty Q() object"):
```

Comment:
Maybe using single quote is more compatible.

Reply:
You're quite right, the coding standards say it should be single quotes here. I've fixed that now.